mod_papers/pubs.html

```diff
@@ -0,0 +1 @@
eight:2.5em;"><br><h2>2011</h2><b>A comparative density functional theory investigation of the mechanical and energetic properties of ZnS</b><br>Feigl, C., Russo, S.P. and Barnard, A.S.<br><a href="http://dx.doi.org/10.1080/08927022.2011.553227">Molecular Simulation <b>37</b> 321-333 (2011)</a><br style="line-height:2.5em;"><b>Nanodiamond induced high-Q resonances in defect-free photonic crystal slabs</b><br>Tomljenovic-Hanic, S., Greentree, A.D., Gibson, B.C., Karle, T.J. and Prawer, S.<br><a href="http://dx.doi.org/10.1364/OE.19.022219">Optics Express <b>19</b> 22219-22226 (2011)</a><br style="line-height:2.5em;"><b>Single photon emission from nanodiamond in tellurite glass</b><br>Gibson, B.C., Henderson, M.R., Ebendorff-Heidepriem, H., Kuan, K., Afshar, V.S., Orwa, J.O., Aharonovich, I., Tomljenovic-Hanic, S., Prawer, S., Monro, T.M. and Greentree, A.D.<br><a href="http://dx.doi.org/10.1109/IQEC-CLEO.2011.6193950">2011 Int. Quantum Electron. Conf., IQEC 2011 and Conf. Lasers and Electro-Optics, CLEO Pacific Rim 2011 Incorporating the Australasian Conf. Optics, Lasers and Spectroscopy and the Australian Conf. <b></b> 721-722 (2011)</a><br style="line-height:2.5em;"><b>Diamond photonics</b><br>Aharonovich, I., Greentree, A.D. and Prawer, S.<br><a href="http://dx.doi.org/10.1038/nphoton.2011.54">Nature Photonics <b>5</b> 397-405 (2011)</a><br style="line-height:2.5em;"><b>A kinetic Monte Carlo study of Pt on Au(111) with applications to bimetallic catalysis</b><br>Zoontjens, P., Grochola, G., Snook, I.K. and Russo, S.P.<br><a href="http://dx.doi.org/10.1088/0953-8984/23/1/015302">Journal of Physics Condensed Matter <b>23</b> 015302 (2011)</a><br style="line-height:2.5em;"><b>Electrostatic-Aharonov-Bohm-like interferometry with adiabatic passage in quantum dots</b><br>Jong, L.M. and Greentree, A.D.<br><a href="http://dx.doi.org/10.1063/1.3630166">AIP Conference Proceedings <b>1363</b> 137-140 (2011)</a><br style="line-height:2.5em;"><b>Lasing, trapping states, and multistability in a circuit quantum electrodynamical analog of a single-atom injection maser</b><br>Marthaler, Michael,  Leppaekangas, Juha and  Cole, Jared H.<br><a href="http://dx.doi.org/10.1103/PhysRevB.83.180505">Physical Review B <b>83</b> 180505 (2011)</a><br style="line-height:2.5em;"><b>Entangling microscopic defects via a macroscopic quantum shuttle</b><br>Grabovskij, G. J.,  Bushev, P.,  Cole, J. H.,  Mueller, C.,  Lisenfeld, J.,  Lukashenko, A. and  Ustinov, A. V.<br><a href="http://dx.doi.org/10.1088/1367-2630/13/6/063015">New Journal of Physics <b>13</b> 063015 (2011)</a><br style="line-height:2.5em;"><b>Ultralow-power spectroscopy of a rare-earth spin ensemble using a superconducting resonator</b><br>Bushev, P.,  Feofanov, A. K.,  Rotzinger, H.,  Protopopov, I.,  Cole, J. H.,  Wilson, C. M.,  Fischer, G.,  Lukashenko, A. and  Ustinov, A. V.<br><a href="http://dx.doi.org/10.1103/PhysRevB.84.060501">Physical Review B <b>84</b> 060501 (2011)</a><br style="line-height:2.5em;">
```

Comment:
The same with this one I'm fairly certain. Both of these files are output but are not required for the module to run at all yeah?

Reply:
yes, you are right

Reply:
es you are right

Reply:
I wasn't really clear here. I meant that this file does not require to be in the repository. It's a file that our list needs to function, but isn't required for other users of the module&mdash;the file is generated buy the script and used at a later date, sure, just not required in the repository.